pkgs/os-specific/linux/open-isns/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, openssl, fetchurl }:
+let
+  pname = "open-isns-0.95";
+in stdenv.mkDerivation {
+  name = "${pname}";
+
+  src = fetchurl {
+    url = "https://github.com/gonzoleeman/open-isns/archive/v0.95.tar.gz";
+    sha256 = "0anv1jl2swfbird5gzxdhdxfxp9gza8w8lpc04lhnpklzngyh21m";
+  };
+
+  propagatedBuildInputs = [ openssl ];
+  outputs = ["out" "lib" ];
+  outputInclude = "lib";
+
+  configureFlags = ["--prefix=/"];
```

Comment:
I dropped the --prefix="/" which read adds the /nix/store paths via configure. Unfortunately that's not 100% complete, so I had to pass in etcdir and vardir in the installPhase.